src/main/webapp/timestamps.js

```diff
@@ -39,3 +39,13 @@ function timestampToString(timestampMs) {
 function secondsToMilliseconds(seconds) {
   return Math.round(seconds * MILLISECONDS_PER_SECOND);
 }
+
+/**
+ * Returns {String} A time range from
```

Comment:
Is putting `{String}` here recommended by the style guide? never seen it before

Reply:
I believe so: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

```
In methods and named functions, parameter and return types must be documented, 
except in the case of same-signature @overrides, where all types are omitted. 
```